package.json

```diff
@@ -5,6 +5,8 @@
   "scripts": {
     "build": "grafana-toolkit plugin:build",
     "test": "jest --notify --watch",
+    "test:coverage": "jest --coverage",
+    "test:coverage:changes": "jest --coverage --changedSince=origin/main",
```

Comment:
I find these commands quite handy... But they are not needed for this PR, 
happy to make this it's own PR if preferred?